templates/component/src/index.js

```diff
@@ -0,0 +1,30 @@
+import { tag } from '@storefront/core';
+
+@tag('<%= slug %>', require('./<%= slug %>.html'), require('./<%= slug %>.scss'))
+export class <%= sanitizedName %> {
+  onBeforeMount() {
```

Comment:
Don't forget about `init() {}` 😄  should be at the top

Reply:
Good call, will update!

Reply:
Implemented, resolving.